examples/flash/main.py

```diff
@@ -24,13 +26,34 @@
 class FlashInterface(BoxLayout):
 
     def turn_on(self):
-        flash.on()
+        try:
+            flash.on()
+        except NotImplementedError:
+            self.ErMsg = "Feature under development for this platform!"
+            popup = Popup(title="Error!",
+                          content=Label(text=self.ErMsg),
+                          size_hint=(None, None), size=(350, 350))
+            popup.open()
 
     def turn_off(self):
-        flash.off()
+        try:
+            flash.off()
+        except NotImplementedError:
+            self.ErMsg = "Feature under development for this platform!"
+            popup = Popup(title="Error!",
+                          content=Label(text=self.ErMsg),
+                          size_hint=(None, None), size=(350, 350))
+            popup.open()
 
     def release(self):
-        flash.release()
+        try:
+            flash.release()
+        except NotImplementedError:
+            self.ErMsg = "Feature under development for this platform!"
+            popup = Popup(title="Error!",
+                          content=Label(text=self.ErMsg),
+                          size_hint=(None, None), size=(350, 350))
+            popup.open()
```

Comment:
@kiok46 I will reflect the following changes according to your feeds in my next commit. :)

* Make a separate class for popup so that it is not implemented three time separately and reflect the same to other files.
* Change error message to "Feature not implemented for this platform!"